Index: /trunk/boost/variant/variant.hpp
===================================================================
--- /trunk/boost/variant/variant.hpp	(revision 85079)
+++ /trunk/boost/variant/variant.hpp	(revision 85080)
@@ -1981,5 +1981,5 @@
     private: // helpers, for internal visitor interface (below)
 
-        template <typename RhsT, typename B1, typename B2>
+        template <typename RhsT, typename B2>
         void assign_impl(
               RhsT& rhs_content
